pipeline_dp/budget_accounting.py

```diff
@@ -116,7 +124,9 @@ def __init__(self, total_epsilon: float, total_delta: float):
     def request_budget(self,
                        noise_kind: NoiseKind,
                        sensitivity: float = 1,
-                       weight: float = 1) -> MechanismSpec:
+                       weight: float = 1,
```

Comment:
Please raise NotImplementedError exception when count != 1 and noise_standard_deviation != None

Reply:
Done